flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/FileSystemJobResultStoreTest.java

```diff
@@ -62,11 +62,11 @@ public void setupTest() throws IOException {
     }
 
     @Test
-    public void testValidEntryPathCreation() {
```

Comment:
Renaming of the methods is not really necessary in the scope of FLINK-28902. The `Internal` suffix doesn't add any value to the method names, I feel. This applies to the other method name changes as well. 

The motivation around renaming the file is to make it being picked up by the surefire plugin. Therefore, renaming the class and the corresponding file is good enough. :+1: